apophis/core/readers.py

```diff
@@ -8,3 +9,12 @@ def fetch_config():
         return config
     except IOError:
         return {'error': 'config does not exist'}
+
+
+async def get_reminders(context) -> List[Any]:
```

Comment:
Why is this in `readers.py`

Reply:
I can refactor and move this to the reminders process, or command, when I first implemented the command, I was storing the reminders in a `.json` file.